frontend/resourceadm/pages/ResourceDashboardPage/ResourceDashboardPage.tsx

```diff
@@ -99,26 +100,39 @@ export const ResourceDashboardPage = (): React.ReactNode => {
           })}
         </Heading>
         <div className={classes.topRightWrapper}>
+          {shouldDisplayFeature('resourceAccessLists') && (
+            <>
+              <StudioButton
```

Comment:
The ```StudioButton``` component has the two props ```icon``` and ```iconPlacement```. I suggest using them as it will position and size the icons correctly on the buttons 😄